doc/en/user/source/extensions/vectortiles/options.rst

```diff
@@ -0,0 +1,108 @@
+.. _vectortiles.options:
+
+Vector Tiles Generation Options
+-------------------------------
+
+The content of the vector tiles generated by this output format is driven by the
+style used in the GetMap/GetTile request.
+
+The style can be pretty basic, the vector tile output format considers only the following:
+
+* Rules scale denominators, and the rule filters, control which features are included in the output
+  (e.g., in a road layer, one can include only highways at low zoom levels)
+* Feature type style and rule vendor options can provide more fine grained control over the output. 
+  Inclusion of options in the rule is recommended only when trying to apply the options in a  scale dependent way.
+
+In particular, the following vendor options can be included:
+
+.. list-table::
+   :widths: 20 80
+   :header-rows: 1
+
+   * - Option
+     - Description
+   * - vt-attributes
+     - A comma separated list of attributes to be included in the output. Reducing the attributes helps control the tile size.
+   * - vt-coalesce
+     - ``true``/``false``, enable feature geometry merging, when attribute values are equal. This can greatly reduce the tile size,
+       especially when coupled when a small attribute list, which helps finding more attribute duplicates.
+       To perform duplicate detection the code will run a "sort" on the attributes, if the source is a database, best to keep the
+       indexed ones first in the list.
+   * - vt-labels
+     - ``true``/``false``, enable the generation of a sidecar "<layerName>-labels" layer with label points. Mean to be used only
+       for polygon geometries, as vector tile clients often lack the ability to compute a suitable label point.
+   * - vt-label-attributes
+     - A comma separate list of attributes to include in the label layer. Typically, only the attributes needed to create the label.
+       Will generate features only if at least one of the attribute values is not null, that is, if there is a chance of creating a label.
```

Comment:
A comma-separated list of attributes to include in the label layer. Typically, only the attributes needed to create the label will be included. Features will be generated only if at least one of the attribute values is not null, meaning there is a chance of creating a label.